data/presets/railway/switch.json

```diff
@@ -1,5 +1,8 @@
 {
     "icon": "temaki-junction",
+    "fields": [
+        "ref_switch"
+    ],
```

Comment:
Side note: 

It looks like `railway:switch` is as commonly used (numbers: https://taginfo.openstreetmap.org/tags/railway%3Dswitch#combinations) as `ref`. Maybe something for a follow up PR? 

https://wiki.openstreetmap.org/wiki/DE:Tag:railway%3Dswitch

Reply:
@tordans Good idea, thanks! Follow-up PR: #1084